[2021-10-16 13:51:04,096] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: fourth_dag.get_gross_year_0 scheduled__2021-10-16T13:48:00+00:00 [queued]>
[2021-10-16 13:51:04,124] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: fourth_dag.get_gross_year_0 scheduled__2021-10-16T13:48:00+00:00 [queued]>
[2021-10-16 13:51:04,126] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-10-16 13:51:04,126] {taskinstance.py:1242} INFO - Starting attempt 1 of 1
[2021-10-16 13:51:04,126] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-10-16 13:51:04,230] {taskinstance.py:1262} INFO - Executing <Task(PythonOperator): get_gross_year_0> on 2021-10-16 13:48:00+00:00
[2021-10-16 13:51:04,238] {standard_task_runner.py:52} INFO - Started process 3312 to run task
[2021-10-16 13:51:04,248] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'fourth_dag', 'get_gross_year_0', 'scheduled__2021-10-16T13:48:00+00:00', '--job-id', '717', '--raw', '--subdir', 'DAGS_FOLDER/fourth_dag.py', '--cfg-path', '/tmp/tmp5666g2ey', '--error-file', '/tmp/tmpanjejnpa']
[2021-10-16 13:51:04,249] {standard_task_runner.py:77} INFO - Job 717: Subtask get_gross_year_0
[2021-10-16 13:51:04,331] {logging_mixin.py:109} INFO - Running <TaskInstance: fourth_dag.get_gross_year_0 scheduled__2021-10-16T13:48:00+00:00 [running]> on host e03fb9e01fd8
[2021-10-16 13:51:04,420] {taskinstance.py:1414} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=fourth_dag
AIRFLOW_CTX_TASK_ID=get_gross_year_0
AIRFLOW_CTX_EXECUTION_DATE=2021-10-16T13:48:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-10-16T13:48:00+00:00
[2021-10-16 13:51:04,435] {python.py:152} INFO - Done. Returned value was: ['$42.05 ', '2010\n']
[2021-10-16 13:51:04,472] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=fourth_dag, task_id=get_gross_year_0, execution_date=20211016T134800, start_date=20211016T135104, end_date=20211016T135104
[2021-10-16 13:51:04,503] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-10-16 13:51:04,571] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
